src/django_browser_reload/views.py

```diff
@@ -46,7 +48,10 @@ def trigger_reload_soon() -> None:
     if reload_timer is not None:
         reload_timer.cancel()
 
-    reload_timer = threading.Timer(RELOAD_DEBOUNCE_TIME, should_reload_event.set)
+    def set_should_reload_event() -> None:
+        asyncio.get_event_loop().run_until_complete(should_reload_event.set())
```

Comment:
this runs in a separate thread, it shouldn't need an event loop?

Reply:
leave it as a threading event. I guess the async version will need to poll the event with an async sleep

Reply:
Ok, that make sense. I was too happy to see I could get rid of the `PING_DELAY`, but you’re right, that will keep things simpler.

Reply:
You can't get rid of `PING_DELAY`, that's required to ensure the pings are sent to keep the connection open. Since `asyncio.Event` doesn't support wait without a timeout, we can't use it.